### ISSUE TITLE
stepmania: failed patch 11_all_system-pcre.patch

### ISSUE THREAD

**Issue by latot** [gentoo-zh/overlay#247]
Hi, trying to recompile stepmania i recived the error with the patch:

```
***** 11_all_system-pcre.patch *****
PWD: /var/tmp/portage/games-arcade/stepmania-5.9999/work/stepmania-5.9999

====================================

PATCH COMMAND:  patch -p0 -g0 -E --no-backup-if-mismatch  < '/var/lib/layman/gentoo-zh/games-arcade/stepmania/files/11_all_system-pcre.patch'

====================================
can't find file to patch at input line 4
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff -ru stepmania-5.9999.old/configure.ac stepmania-5.9999/configure.ac
|--- stepmania-5.9999.old/configure.ac  2015-01-15 23:43:14.480532494 +0300
|+++ stepmania-5.9999/configure.ac  2015-01-15 23:54:43.906807439 +0300
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored
can't find file to patch at input line 30
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|Только в stepmania-5.9999.old: .configure.ac.swp
|diff -ru stepmania-5.9999.old/src/Makefile.am stepmania-5.9999/src/Makefile.am
|--- stepmania-5.9999.old/src/Makefile.am   2015-01-15 23:43:13.573853213 +0300
|+++ stepmania-5.9999/src/Makefile.am   2015-01-15 23:58:23.859879075 +0300
--------------------------
No file to patch.  Skipping patch.
3 out of 3 hunks ignored
can't find file to patch at input line 65
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff -ru stepmania-5.9999.old/src/RageUtil.cpp stepmania-5.9999/src/RageUtil.cpp
|--- stepmania-5.9999.old/src/RageUtil.cpp  2015-01-15 23:43:13.653854325 +0300
|+++ stepmania-5.9999/src/RageUtil.cpp  2015-01-16 00:01:17.248956597 +0300
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored

patch program exited with status 1
====================================

PATCH COMMAND:  patch -p1 -g0 -E --no-backup-if-mismatch  < '/var/lib/layman/gentoo-zh/games-arcade/stepmania/files/11_all_system-pcre.patch'

====================================
checking file configure.ac
Hunk #1 succeeded at 388 (offset 6 lines).
checking file src/Makefile.am
Hunk #1 FAILED at 479.
Hunk #2 FAILED at 515.
Hunk #3 FAILED at 763.
3 out of 3 hunks FAILED
checking file src/RageUtil.cpp
Hunk #1 FAILED at 1342.
1 out of 1 hunk FAILED

patch program exited with status 1
====================================

PATCH COMMAND:  patch -p2 -g0 -E --no-backup-if-mismatch  < '/var/lib/layman/gentoo-zh/games-arcade/stepmania/files/11_all_system-pcre.patch'

====================================
can't find file to patch at input line 4
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff -ru stepmania-5.9999.old/configure.ac stepmania-5.9999/configure.ac
|--- stepmania-5.9999.old/configure.ac  2015-01-15 23:43:14.480532494 +0300
|+++ stepmania-5.9999/configure.ac  2015-01-15 23:54:43.906807439 +0300
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored
checking file Makefile.am
Hunk #1 FAILED at 479.
Hunk #2 FAILED at 515.
Hunk #3 FAILED at 763.
3 out of 3 hunks FAILED
can't find file to patch at input line 65
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff -ru stepmania-5.9999.old/src/RageUtil.cpp stepmania-5.9999/src/RageUtil.cpp
|--- stepmania-5.9999.old/src/RageUtil.cpp  2015-01-15 23:43:13.653854325 +0300
|+++ stepmania-5.9999/src/RageUtil.cpp  2015-01-16 00:01:17.248956597 +0300
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored

patch program exited with status 1
====================================

PATCH COMMAND:  patch -p3 -g0 -E --no-backup-if-mismatch  < '/var/lib/layman/gentoo-zh/games-arcade/stepmania/files/11_all_system-pcre.patch'

====================================
can't find file to patch at input line 4
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff -ru stepmania-5.9999.old/configure.ac stepmania-5.9999/configure.ac
|--- stepmania-5.9999.old/configure.ac  2015-01-15 23:43:14.480532494 +0300
|+++ stepmania-5.9999/configure.ac  2015-01-15 23:54:43.906807439 +0300
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored
can't find file to patch at input line 30
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|Только в stepmania-5.9999.old: .configure.ac.swp
|diff -ru stepmania-5.9999.old/src/Makefile.am stepmania-5.9999/src/Makefile.am
|--- stepmania-5.9999.old/src/Makefile.am   2015-01-15 23:43:13.573853213 +0300
|+++ stepmania-5.9999/src/Makefile.am   2015-01-15 23:58:23.859879075 +0300
--------------------------
No file to patch.  Skipping patch.
3 out of 3 hunks ignored
can't find file to patch at input line 65
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff -ru stepmania-5.9999.old/src/RageUtil.cpp stepmania-5.9999/src/RageUtil.cpp
|--- stepmania-5.9999.old/src/RageUtil.cpp  2015-01-15 23:43:13.653854325 +0300
|+++ stepmania-5.9999/src/RageUtil.cpp  2015-01-16 00:01:17.248956597 +0300
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored

patch program exited with status 1
====================================

PATCH COMMAND:  patch -p4 -g0 -E --no-backup-if-mismatch  < '/var/lib/layman/gentoo-zh/games-arcade/stepmania/files/11_all_system-pcre.patch'

====================================
can't find file to patch at input line 4
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff -ru stepmania-5.9999.old/configure.ac stepmania-5.9999/configure.ac
|--- stepmania-5.9999.old/configure.ac  2015-01-15 23:43:14.480532494 +0300
|+++ stepmania-5.9999/configure.ac  2015-01-15 23:54:43.906807439 +0300
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored
can't find file to patch at input line 30
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|Только в stepmania-5.9999.old: .configure.ac.swp
|diff -ru stepmania-5.9999.old/src/Makefile.am stepmania-5.9999/src/Makefile.am
|--- stepmania-5.9999.old/src/Makefile.am   2015-01-15 23:43:13.573853213 +0300
|+++ stepmania-5.9999/src/Makefile.am   2015-01-15 23:58:23.859879075 +0300
--------------------------
No file to patch.  Skipping patch.
3 out of 3 hunks ignored
can't find file to patch at input line 65
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff -ru stepmania-5.9999.old/src/RageUtil.cpp stepmania-5.9999/src/RageUtil.cpp
|--- stepmania-5.9999.old/src/RageUtil.cpp  2015-01-15 23:43:13.653854325 +0300
|+++ stepmania-5.9999/src/RageUtil.cpp  2015-01-16 00:01:17.248956597 +0300
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored

patch program exited with status 1
```

thx.

**Comment by biergaizi** [comment]
The Use-System-PCRE patch was merged to upstream, that's why it fails. Easy to fix.

**Comment by biergaizi** [comment]
Fixed in c5dd27d0b17b963fa8cae09be9d1fdd86ab2f1a5.
